source/common/runtime/runtime_features.cc

```diff
@@ -77,7 +77,10 @@ RUNTIME_GUARD(envoy_reloadable_features_successful_active_health_check_uneject_h
 RUNTIME_GUARD(envoy_reloadable_features_test_feature_true);
 RUNTIME_GUARD(envoy_reloadable_features_thrift_allow_negative_field_ids);
 RUNTIME_GUARD(envoy_reloadable_features_thrift_connection_draining);
+RUNTIME_GUARD(envoy_reloadable_features_token_passed_entirely);
 RUNTIME_GUARD(envoy_reloadable_features_uhv_allow_malformed_url_encoding);
+RUNTIME_GUARD(envoy_reloadable_features_uhv_preserve_url_encoded_case);
```

Comment:
These runtime keys looks like result of a bad merge. Please remove them, since they are not part of this change.